package.json

```diff
@@ -461,6 +461,16 @@
         "command": "terraform.cloud.workspaces.refresh",
         "title": "Refresh",
         "icon": "$(refresh)"
+      },
+      {
+        "command": "terraform.cloud.runs.refresh",
+        "title": "Refresh",
+        "icon": "$(refresh)"
+      },
+      {
+        "command": "terraform.cloud.run.view",
```

Comment:
Command names have to be unique across the entirety of VS Code, so to prevent confusion and collision I would like to name this something more descriptive than `view`. We have the word `view` everywhere so it's hard to determine what this command will do just by reading the name.

How about `terraform.cloud.run.viewInBrowser`?

The title can remain the same as from the user POV they see a globe and view and that implies internet.